raster_messenger/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM maven:3.6.0-jdk-8-slim
+LABEL maintainer="Julian Kaeflein<julian.kaeflein@geomer.de>"
+
+ENV UM_SERVER="host.docker.internal:9876"
+ENV UM_CHANNEL="raster_data"
+ENV TEST_REGION="TestRegion"
+ENV TEST_POLLUTANT="TestSchad"
+
+RUN apt-get update && apt-get -y install -qq cron
```

Comment:
Necessary to install `cron`?

Reply:
No, just the usual: 

&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;Good
&ensp;&ensp;&ensp;/&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;\\
 Fast &ensp; &ensp; \__me\__ &ensp;&ensp;  Cheap

Will commit after all comments resolved.